data/mods/Aftershock/mobs/monster_faction.json

```diff
@@ -7,6 +7,12 @@
     "by_mood": [ "insect", "small_animal" ],
     "hate": [ "fungus" ]
   },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "WraitheonRobotics",
```

Comment:
Any particular attitude towards slime?

Reply:
Slime is unlikely to make the cut in the new aftershock setting.